staging/src/k8s.io/kubelet/pkg/apis/dra/v1/api.proto

```diff
@@ -72,9 +72,13 @@ message Device {
     // The device itself. Required.
     string device_name = 3;
 
+    // The share ID of device.
+    // Optional.
+    optional string share_id = 4;
+
     // A single device instance may map to several CDI device IDs.
     // None is also valid.
-    repeated string cdi_device_ids = 4;
+    repeated string cdi_device_ids = 5;
```

Comment:
leave the tag numbers on the existing fields as-is, and allocate any new fields *new* numbers